designs/lily/src/waistband.mjs

```diff
@@ -0,0 +1,121 @@
+import { back } from './back.mjs'
+import { front } from './front.mjs'
+// do not import from Paco since that would use Paco's back to define the waistband
```

Comment:
remove these 2 lines

Reply:
Actually, this is 99% paco's waistband. So why don't you start from that and just remove the eyelets?

Reply:
Note that you'll need to add paco to the peer dependencies too

Reply:
> Actually, this is 99% paco's waistband. So why don't you start from that and just remove the eyelets?

I believe I tried that (a long time ago) but reverted it since (according to the note) that would use `paco`'s back to define the waistband (length), and `paco`'s waist circumference is adjusted to fit around the seat without stretching.